src/ui/wxWidgets/PasswordPolicyDlg.cpp

```diff
@@ -763,6 +775,8 @@ void PasswordPolicyDlg::OnUseNamedPolicy( wxCommandEvent& event )
     m_pwpUCSpin->Enable(m_pwpUseUpperCtrl->GetValue());
     m_pwpDigSpin->Enable(m_pwpUseDigitsCtrl->GetValue());
     m_pwpSymSpin->Enable(m_pwpSymCtrl->GetValue());
+    m_OwnSymbols->Enable(m_pwpSymCtrl->GetValue());
+    FindWindow(ID_RESET_SYMBOLS)->Enable(m_pwpSymCtrl->GetValue());
```

Comment:
Is it possible for `FindWindow` to return null in some cases?

Reply:
Good question. Since the window (button) associated with this id is created unconditionally along with the rest of the dialog's control's, I'd say no. Or, more accurately, if FindWindow() won't find this control, we have bigger problems...